dev_docs/operations/packages_idm.mdx

```diff
@@ -51,19 +51,16 @@ Package types allow us to have many different packages, pre-defined build tasks,
 : These packages can be imported from all other packages.
 
 `shared-browser`
-: These packages can be imported from `shared-browser` and `plugin-browser` packages. `shared-browser` packages may include Storybooks.
+: These packages can be imported from `shared-browser` and `public` directories within `plugin` packages. `shared-browser` packages may include Storybooks.
 
 `shared-server`
-: These packages can be imported from `shared-server` and `plugin-server` packages.
+: These packages can be imported from `shared-server` and `server` directories within `plugin` packages.
 
 `shared-scss`
-: These packages can be imported by `shared-browser` and `plugin-browser` packages, and expose an `index.scss` file to consumers instead of an `index.ts` file.
+: These packages can be imported by `shared-browser` and `public` directories within `plugin` packages, and expose an `index.scss` file to consumers instead of an `index.ts` file.
 
-`plugin-browser`
-: These packages expose types to other packages via a root `types.ts` file.  Module IDs must end with `-plugin-browser`. Consumers must use `import type` statements.
-
-`plugin-server`
-: These packages expose types to other packages via a root `types.ts` file. Module IDs must end with `-plugin-server`. Consumers must use `import type` statements.
+`plugin`
```

Comment:
For now I'm not sure I see the exact value of putting the work in to split up plugins into `server` and `browser` packages. For now the package type is just "plugin" and the source_classifier will determine if something is server/browser/common source using the existing complex rules going forward. Ideally we'll naturally move most code out of plugins and it will be easier in the future to breakup plugins, if it still makes sense.